frontend/src/index.css

```diff
@@ -13,82 +11,67 @@ body {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     background-color: white;
+    
+}
+
+.row{
+    margin-right:0rem;
 }
 
 .input-group-text {
-    background-color: white;
+    background-color: #F5F5F5;
     border-radius: 20rem;
     color: #495057bf;
+    padding: 0rem .75rem;
 }
 
 .form-control {
     border-radius: 20rem;
+    background-color: #F5F5F5;
+    font-size:0.96rem;
 }
 
-.todo-title {
-    cursor: pointer;
-}
-
-.tab-list>span {
-    padding: 5px 8px;
-    border: 1px solid #282c34;
-    border-radius: 10px;
-    margin-right: 5px;
-    cursor: pointer;
-}
-
-.tab-list>span.active {
-    background-color: #282c34;
-    color: #ffffff;
-}
-
-.btn-light {
-    background-color: #e0e0e075;
-    margin-bottom: 7px;
-    border-radius: 0px;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    box-shadow: 1px 1px #80808000;
-}
-
-.row {
-    margin-right: 0px;
-    margin-left: 0px;
-    height: 100vh;
+.title{
+    font-weight: bold;
 }
 
-.h3 {
-    font-family: 'Libre Baskerville', serif;
+.clientadvisorBackground{
+    background-color: #CCCCCC;
 }
 
-.p-4 {
-    padding: 5rem!important;
+.loginBackground{
```

Comment:
lets try to make all our css class names hyphenated and not camelCase (i see a few diff ones in here so lets just change them all earlier than later)